org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/Filters.java

```diff
@@ -43,6 +43,7 @@ public static IFilter all() {
 				new ExhaustiveSwitchFilter(), //
 				new RecordPatternFilter(), //
 				new AnnotationGeneratedFilter(), new KotlinGeneratedFilter(),
+				new KotlinEnumFilter(), //
```

Comment:
Why do we need a `//` here?

Reply:
Otherwise Eclipse JDT Formatter will join lines and the change will look like

```diff
                                new AnnotationGeneratedFilter(), new KotlinGeneratedFilter(),
-                               new KotlinLateinitFilter(), new KotlinWhenFilter(),
-                               new KotlinWhenStringFilter(),
+                               new KotlinEnumFilter(), new KotlinLateinitFilter(),
+                               new KotlinWhenFilter(), new KotlinWhenStringFilter(),
                                new KotlinUnsafeCastOperatorFilter(),
```